### PR TITLE
add Linux Mint 20.1 (X86_64) support

### PR DESCRIPTION
**Pull request by Danile71**

### CODE REVIEW

File: inference-engine/cmake/dependencies.cmake [hunk 1]
```diff
@@ -250,6 +250,9 @@ if (ENABLE_OPENCV)
             elseif (LINUX_OS_NAME STREQUAL "Ubuntu 20.04" AND X86_64)
                 set(OPENCV_SUFFIX "ubuntu20")
                 set(OPENCV_HASH "2b7808d002864acdc5fc0b19cd30dadc31a37cc267931cad605f23f2383bfc21")
+            elseif (LINUX_OS_NAME STREQUAL "LinuxMint 20.1" AND X86_64)
```
Comment (ilya-lavrenov) [inline inference-engine/cmake/dependencies.cmake:253]: can we just change the previous condition to be:
```cmake
elseif ((LINUX_OS_NAME STREQUAL "Ubuntu 20.04" OR LINUX_OS_NAME STREQUAL "LinuxMint 20.1") AND X86_64)
                set(OPENCV_SUFFIX "ubuntu20")
                set(OPENCV_HASH "2b7808d002864acdc5fc0b19cd30dadc31a37cc267931cad605f23f2383bfc21")
elseif(...)
```

Comment (Danile71) [reply inference-engine/cmake/dependencies.cmake:253]: Later, you can set different hash for Ubuntu and Mint, or not?

Comment (ilya-lavrenov) [reply inference-engine/cmake/dependencies.cmake:253]: If these OSes are compatible now, no needs to provide different packages in future I suppose

Comment (Danile71) [reply inference-engine/cmake/dependencies.cmake:253]: ok)

Comment (ilya-lavrenov) [reply inference-engine/cmake/dependencies.cmake:253]: Could you please update a patch?

**Review by ilya-lavrenov — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**